lib/type/any.ex

```diff
@@ -8,13 +8,31 @@ defmodule Litmus.Type.Any do
   @type t :: %__MODULE__{
           required: boolean
         }
+
+  @spec add_required_errors(map, binary, t) :: {:ok, map} | {:error, String.t()}
+  def add_required_errors(params, field, %Litmus.Type.Any{required: true}) do
+    if Map.has_key?(params, field) do
+      {:ok, params}
+    else
+      {:error, "#{field} is required"}
+    end
+  end
+
+  def add_required_errors(params, _field, %Litmus.Type.Any{required: false}), do: {:ok, params}
+
+  def add_required_errors(_params, field, %Litmus.Type.Any{required: _}) do
+    {:error, "Incorrect schema defined for #{field}"}
```

Comment:
You could make this even easier for someone to debug with something along the lines of "Any.required must be a boolean".